functions/enqueues.php

```diff
@@ -2,16 +2,39 @@
 /**
  * Enqueue scripts and styles.
  */
+function bootscout_theme_preconnect() {
+    echo '<link rel="preconnect" href="https://fonts.googleapis.com">' . PHP_EOL;
+    echo '<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>' . PHP_EOL;
+}
+
 function bootscout_theme_scripts() {
 	// Styles
-
-	wp_register_style('BootstrapIcons', 'https://cdn.jsdelivr.net/npm/bootstrap-icons@1.7.0/font/bootstrap-icons.css', false, '1.7.0', null);
+    wp_register_style(
+        'Nunito+Sans',
+        'https://fonts.googleapis.com/css2?family=Nunito+Sans:ital,wght@0,400;0,600;0,700;0,900;1,400;1,600;1,700;1,900&display=swap',
+        false,
+        null,
+        null
+    );
+    wp_enqueue_style('Nunito+Sans');
+	wp_register_style(
+	    'BootstrapIcons',
+        'https://cdn.jsdelivr.net/npm/bootstrap-icons@1.7.0/font/bootstrap-icons.css',
+        false,
+        null,
```

Comment:
Unset version param here since it's a CDN file with the version in the URI so it doesn't need cache busting.